Mu2eG4/src/Mu2eG4SteppingAction.cc

```diff
@@ -155,6 +175,13 @@ namespace mu2e {
       }
     }
 
+    //for killing low momentum photon conversions in materials 
+    if(gammaFilterAccepted_ < 0 || //if < 0 kill entire event without checking tracks
+       (gammaFilterMinDaughterEnergy_ > 0. && gammaFilterMaxGammaEnergy_ > 0. && //parameters are set, if not ignore this check
+	((gammaFilterMinDaughterEnergy_ > gammaFilterMaxGammaEnergy_) || //kill all events if min energy > maximum photon energy
+	 (!gammaFilterAccepted_ && killLowMomentumGammaDaughters(track))))) //kill if not accepted yet and fails check
+      killTrack(track, ProcessCode::mu2eLowEnergyGammaKilled, fStopAndKill);
```

Comment:
I would suggest adding {} around the function, to avoid potential if scope problems in future